user-documentation/workshops/recipe-authoring.md

```diff
@@ -526,23 +478,9 @@ You can also run best practice recipes against your rewrite recipe module, to re
 
 You can apply these recommendations to your recipes by running the following command:
 
-{% tabs %}
-{% tab title="Gradle" %}
-{% code overflow="wrap" %}
-```bash
-./gradlew rewriteRun -Drewrite.activeRecipe=org.openrewrite.recipes.OpenRewriteBestPractices
-```
-{% endcode %}
-{% endtab %}
-
-{% tab title="Maven" %}
-{% code overflow="wrap" %}
 ```bash
-mvn -U org.openrewrite.maven:rewrite-maven-plugin:run -Drewrite.recipeArtifactCoordinates=org.openrewrite.recipe:rewrite-recommendations:RELEASE -Drewrite.activeRecipes=org.openrewrite.recipes.OpenRewriteBestPractices
+mod run /path/to/your/recipe --recipe="org.openrewrite.recipes.OpenRewriteBestPractices"
```

Comment:
Wherever possible I prefer the shorthand instead:
```suggestion
mod run /path/to/your/recipe --recipe=OpenRewriteBestPractices
```

Reply:
Good to know - thanks for the call out.